src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:not(:last-child),
+.stars--3 .stars__star:not(:nth-child(n+4)),
+.stars--2 .stars__star:not(:nth-child(n+3)),
+.stars--1 .stars__star:first-child {
+  background: url(../images/star-active.svg) center no-repeat;
+}
```

Comment:
Try to use loop here: https://github.com/mate-academy/layout_catalog/blob/master/checklist.md [12 point]